deps/ledger-zxlib/dockerized_build.mk

```diff
@@ -297,7 +297,7 @@ define run_announce
 endef
 
 define run_nodejs_test
-	@cd $(TESTS_SPECULOS_DIR) && TEST_SPECULOS_API_PORT=$(1) TEST_SPECULOS_APDU_PORT=$(2) TEST_DEVICE=$(TEST_DEVICE) node $(3) 2>&1 | tee -a ../../speculos-port-$(1).log 2>&1 | perl -lane '$$lines .= $$_ . "\n"; printf qq[%s\n], $$_ if(m~test(Start|Combo|Step|End)~); sub END{ die qq[ERROR: testEnd not detected; test failed?\n] if($$lines !~ m~testEnd~s); }'
```

Comment:
Why are we removing the processing here?

Reply:
Main purpose of post-processing was to detect errors by comparing testBegin's and testEnd's.This was necessary because node's errors were muted (both by being early in the pipe and by - in make call running the command).
Unfortunately, the check here may not find all issues and as we had seen, sometimes, the tests were green when node failed.
The change causes that make fails when node returns an error. Which, I believe is the think we should be doing.

Besides this, the script allowed to see progress while tests are running in greater detail. This is useful, but I believe there are better ways to do this. After I split transaction tests into smaller batches, this will be almost a non-issue. Thus, for simplicity, I decided to remove it altogether.